test/vagrant-local-start.sh

```diff
@@ -20,6 +20,12 @@ else
     unset SERVER_VERSION
 fi
 
+if [[ "$NFS" != "0" ]]; then
```

Comment:
This ip range are from [here](https://github.com/cilium/cilium/blob/master/test/Vagrantfile#L225-L228)